### ISSUE TITLE
Progress report?

### ISSUE THREAD

**Issue by mor10** [octokit/auth-oauth-user-client.js#7]
Any news on if/when this feature will be built out? Having a viable user client solution for octokit will remediate the problem of developers having to cook their own solution.

**Comment by baoshan** [comment]
I encourage you to give [this](#8) a try. Please let me know if you have any question.

**Comment by QSC-Eugene** [comment]
Any movement on this for 2023?

**Comment by gr2m** [comment]
not really. We laid out what needs to be done here: https://github.com/octokit/octokit.js#oauth-for-browser-apps
and there is https://github.com/baoshan/octokit-auth-oauth-user-client.js

**Comment by ghost** [comment]
Any news about this repo?
